tools/icu/README.md

```diff
@@ -1,4 +1,15 @@
-# Notes about the icu directory.
+# Notes about the `tools/icu` subdirectory
+
+This directory contains tools, data, and information about the [http://icu-project.org](ICU) (International Components for Unicode) integration. ICU is used to provide Intl functionality.
```

Comment:
Not just `Intl.*` — it‘s also used for e.g. RegExp Unicode property escapes.

Reply:
@mathiasbynens Would just using `internationalization functionality` instead of `Intl functionality` sound good to you?

Reply:
Yeah, that sounds good!

Reply:
(dramatic pause) :+1: